airbyte-server/src/main/resources/application.yml

```diff
@@ -45,6 +45,9 @@ airbyte:
     url: ${WEBAPP_URL:}
   workspace:
     root: ${WORKSPACE_ROOT}
+  protocol:
+    min-version: ${AIRBYTE_PROTOCOL_VERSION_MIN:`0.0.0`}
```

Comment:
I don't think that we want the ticks in here.  Micronaut will create a string with those in it.  I suspect we just want `0.0.0`, etc.